webapp/locales/en.json

```diff
@@ -373,6 +373,11 @@
     "questions": "Any Questions or concerns, send an e-mail to",
     "title": "Human Connection is under maintenance"
   },
+  "modals": {
+    "deleteUser": {
+      "created": "Created"
+    }
+  },
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-03-05T09:54:09Z" title="Thursday, March 5th 2020, 10:54:09 am +01:00">Mar 5, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -639,8 +639,11 @@
     "deleteUserAccount": {
       "accountDescription": "Be aware that your Posts and Comments are important to our community. If you still choose to delete them, you have to mark them below.",
       "accountWarning": "You CAN'T MANAGE and CAN'T RECOVER your Account, Posts, or Comments after deleting your account!",
+      "accountWarningAdmin": "The account, contributions or comments can NOT BE ADMINISTERED OR RESTORED after deletion!",
+      "accountWarningIsAdmin": "Heads up! You are Admin!!",
       "commentedCount": "Delete my {count} comments",
       "contributionsCount": "Delete my {count} posts",
+      "infoAdmin": "All contributions and comments of the user are additionally deleted!",
```


`… are deleted as well!` ?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-28T13:58:49Z" title="Thursday, May 28th 2020, 3:58:49 pm +02:00">May 28, 2020</time>_

---


This is not done yet!

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-26T15:09:39Z" title="Wednesday, August 26th 2020, 5:09:39 pm +02:00">Aug 26, 2020</time>_

---


Still not done! 🤔